app/services/vendor_api/generate_test_applications_for_provider.rb

```diff
@@ -1,9 +1,9 @@
 module VendorAPI
   class GenerateTestApplicationsForProvider
-    def call(provider:, courses_per_application:, count:, for_ratified_courses: false)
+    def call(provider:, courses_per_application:, count:, for_training_courses: false, for_ratified_courses: false, for_test_provider_courses: false)
       raise ParameterInvalid, 'Parameter is invalid (cannot be zero): courses_per_application' if courses_per_application.zero?
 
-      course_ids = course_list_for_provider(provider, for_ratified_courses).pluck(:id)
+      course_ids = course_list_for_provider(provider, for_training_courses, for_ratified_courses, for_test_provider_courses).pluck(:id)
```

Comment:
At the moment, this will generate `count` application forms, all with the same course choices.

To make it a bit more realistic, we can run this `count` times and enqueue a Sidekiq job for each. I think that's the only was to do this without a bigger refactor.